# Generated by Django 3.1.12 on 2022-02-08 17:56

import tinymce.models
from django.db import migrations, models


class Migration(migrations.Migration):
    dependencies = [
        ('integration', '0042_add_access_revoked_new_error'),
    ]

    operations = [
        migrations.AddField(
            model_name='errorcatalogue',
            name='default_wizard_message',
            field=tinymce.models.HTMLField(blank=True, default='', null=True),
        ),
        migrations.AddField(
            model_name='integrationalert',
            name='wizard_message',
            field=tinymce.models.HTMLField(blank=True, default='', null=True),
        ),
        migrations.AddField(
            model_name='integrationalert',
            name='wizard_error_code',
            field=models.CharField(blank=True, default='', null=True, max_length=50),
        ),
        migrations.RemoveConstraint(
            model_name='integrationalert',
            name='integration_error_re_unique_error_message',
        ),
        migrations.AddConstraint(
            model_name='integrationalert',
            constraint=models.UniqueConstraint(
                fields=(
                    'integration',
                    'error',
                    'error_response_regex',
                    'wizard_error_code',
                ),
                name='integration_error_re_unique_error_message',
            ),
        ),
    ]
